### ISSUE TITLE
There are just videos

### ISSUE THREAD

**Issue by Pascal-0x90** [Epidemic41/Capstone#1]
---

## tl;dr

Good folks of the Cyber Realm! The Security Configuration Management System repo seems to have decided that code is too mainstream and is living the video-only life. But come on, we need the meaty stuff - the code! We can't `git commit` our eyeballs to learn from videos, can we?

---

# Issue: The Missing Code Chronicles

## Where's the Beef?

To be real, I'm feeling like Neo from The Matrix - seeing lots of fancy moving images (cool videos though, not gonna lie), but I'm missing that green raining code. Your project is named the "Security Configuration Management System", and by its name, I expect it to be an ultimate tool to keep our security configurations in check, orderly and neat as a new subnet.

But here's the irony, all we've got are videos. Sure, videos are great for painting the big picture, but when it comes to the nuts and bolts - the actual code - there's an echo. 

## No Spoon, Only Videos

We are not in the Matrix, there's no spoon and apparently, no code. Only videos. As much as I'd like to believe in a world where I can learn Kung Fu (or any other skill, really) instantly, as Neo did, I know that's not gonna happen (until Elon gets Neuralink up and running, that is). 

## Not Trying to `rm -rf /`, But...

I'm not saying this to troll or `rm -rf /` the project. It's just that without the code, it's like a GitHub repo without the "git". 

## Git Gud, Push Code

To get really good, we need to get our hands dirty with code, not just watch videos. That's how we "git gud", isn't it? 

So, fellow keyboard warriors, let's bring this issue to light, put the "git" back in GitHub and "push" some code!

---

P.S. Don't take any of this in a bad way - the project looks promising and I appreciate the hard work behind it. But let's give it some real legs to stand on - let's give it some code!

**Comment by Epidemic41** [comment]
I got code for ya

**Comment by Pascal-0x90** [comment]
Awesome thanks!
